pkg/io/cryptoReader.go

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Let's move this reader to `pkg/crypto/crypto` packages because mainly they will go together all the road with cryptographer's implementation.